src/Microsoft.Extensions.Configuration.UserSecrets/ConfigurationExtensions.cs

```diff
@@ -3,15 +3,19 @@
 
 using System;
 using System.IO;
+using System.Reflection;
 using Microsoft.Extensions.Configuration.UserSecrets;
+using Microsoft.Extensions.Configuration.UserSecrets.Internal;
 using Microsoft.Extensions.FileProviders;
 
 namespace Microsoft.Extensions.Configuration
 {
+    /// <summary>
+    /// Configuration extensions for adding user secrets configuration source.
+    /// </summary>
     public static class ConfigurationExtensions
     {
-        private const string Secrets_File_Name = "secrets.json";
-
+#if NET451 || NETSTANDARD1_5
```

Comment:
The CLI tool was preview, but the config extension was RTM....so we can't just break.